tests/printer/expr/expected/templateLiteral.res.txt

```diff
@@ -73,7 +73,8 @@ module X = %graphql("
   }
 ")
 
-let cn = css`
+let cn =
+  css`
```

Comment:
Not sure why there's a carriage return here.  Something else to investigate.

Reply:
I tried a bunch of things, but wasn't able to fix this.  I'll try to do some more digging next week.

Reply:
I think adding missing location data might solve this. 🤞

Reply:
Adding location data did not fix the issue so I'll need to compare the `Doc.t` trees that are produced by the new code and the old code to get some more info about what may be causing the difference in output.

Reply:
hmm, yeah, must be some `Doc.t` tree difference that trigger new lines differently in `Doc.toString` indeed

Reply:
I was able to diagnose the `Doc.t` tree differences by enabling the `Doc.debug doc;` at the bottom of res_printer.ml and then figuring out which `Doc.text " ="` corresponded to the one in the debug output.  The issue is fixed now. 🙂